Makefile

```diff
@@ -15,13 +15,21 @@ GO_BIN ?= go
 TEST_COMMAND ?= $(GO_BIN) test
 
 # The compute tests can sometimes exceed the default 10m limit.
-TEST_ARGS ?= -timeout 15m ./{cmd,pkg}/...
+TEST_ARGS ?= -timeout 15m ./...
 
 CLI_ENV ?= "development"
 
 GOHOSTOS ?= $(shell go env GOHOSTOS || echo unknown)
 GOHOSTARCH ?= $(shell go env GOHOSTARCH || echo unknown)
-GO_FILES = $(shell find cmd pkg -type f -name '*.go')
+
+ifeq ($(OS), Windows_NT)
+	SHELL = cmd.exe
+	.SHELLFLAGS = /c
+	GO_FILES = $(shell where /r pkg *.go)
+	GO_FILES += $(shell where /r cmd *.go)
```

Comment:
ℹ️ Recursively searching for files is actually quite tricky in PowerShell (the default CI Windows environment), and so I switched to using `cmd.exe` which is the default Terminal on Windows typically (PowerShell is an enhanced shell environment but it's something you have to manually install on Windows). Additionally, the CLI (because of this fact) uses `cmd.exe` when opening a subshell to execute build scripts on Windows, and so it's better for our CI to be consistent with that.